app/models/ems_refresh/refreshers/refresher_relats_mixin.rb

```diff
@@ -349,14 +358,16 @@ def update_relats(target, prev_relats, new_relats)
       do_relat_compare(:folders_to_folders, prev_relats, new_relats) do |f|
         folder = EmsFolder.find(f)
         [ do_disconnect ? Proc.new { |f2| folder.remove_folder(EmsFolder.find(f2)) } : nil, # Disconnect proc
-          Proc.new { |f2| folder.add_folder(EmsFolder.find(f2)) } ]                         # Connect proc
+          Proc.new { |f2| folder.add_folder(EmsFolder.find_by_id(f2)) },                    # Connect proc
```

Comment:
If `find` is currently working, I'd rather not change to the weaker `find_by_id` (same for `EmsCluster` below).